elm.json

```diff
@@ -8,11 +8,11 @@
         "direct": {
             "elm/browser": "1.0.1",
```

Comment:
I can't comment above, but your version is still `0.19.0` - update it to `0.19.1`.